app/collections/show.jsx

```diff
@@ -52,19 +52,20 @@ class CollectionPage extends React.Component {
     }
   }
 
-  canCollaborate = () => {
+  canCollaborate() {
     let canCollaborate;
+    let isOwner;
     if (!this.props.user) {
       canCollaborate = false;
+      isOwner = false;
     } else {
       canCollaborate = this.props.roles.some((role) => {
         const idMatch = (role.links.owner.id === this.props.user.id);
         const isCollaborator = role.roles.includes('collaborator');
-
         return isCollaborator && idMatch;
       });
+      isOwner = this.props.user.id === this.props.owner.id;
     }
```

Comment:
This could be simplified a bit:

```
    let canCollaborate = false;
    let isOwner = false;
    if (this.props.user) {
      canCollaborate = this.props.roles.some((role) => {
        const idMatch = (role.links.owner.id === this.props.user.id);
        const isCollaborator = role.roles.includes('collaborator');
        return isCollaborator && idMatch;
      });
      isOwner = this.props.user.id === this.props.owner.id;
    }
```

Reply:
Cheers. I rewrote that function to make the logic less complicated to follow. If not… conditions always throw me when I'm reading code.

EsLint is working now, too. 👍